README.md

```diff
@@ -33,3 +33,51 @@ Clearinghouse for PDFs used during the submission process.
 - Localstack (https://github.com/localstack/localstack) can be used to simulate S3.
 - Boto3 (https://boto3.amazonaws.com/v1/documentation/api/latest/index.html) should be used to implement the service module that persists preview content and metadata.
 - Moto (https://github.com/spulec/moto) may be used to mock/fake S3 in tests.
+
+
+## Running the submission preview service locally
+
+```bash
+cd /path/to/arxiv-submission-preview
+docker-compose pull    # build the images
+docker-compose up      # start the service group
+```
+
+Give it a few seconds; localstack needs to come up (provides a local S3), and
+a bucket will be created.
+
+
+### Authentication + Authorization
+
+To use the API you will need an auth token with scopes ``preview:read`` and
+``preview:create``. The easiest way to generate one of these is to use the
```

Comment:
My initial instructions specified ``preview:write`` which was wrong and so I was using an auth token that wouldn't work for the simple example `PUT` request.